jvb/pom.xml

```diff
@@ -281,6 +285,16 @@
       <version>2.3.1</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
Can it just inherit this from `jicoco-metrics`?

Reply:
The `/metrics` endpoint uses media type constants such as `TextFormat.CONTENT_TYPE_004`.
Only `simpleclient_common` is needed for that, but we can have those constants in `MetricsContainer` or write them ourselves in the endpoint.